src/scaffolding/graph.ts

```diff
@@ -30,7 +29,7 @@ export async function fetchPublicKeySchema(): Promise<void> {
   const schema = await new SchemaBuilder()
     .withName('dsnp', 'public-key-key-agreement')
     .withModelType('AvroBinary')
-    .withModel('{type:record,name:PublicKey,namespace:org.dsnp,fields:[{name:publicKey,doc:Multicodec public key,type:bytes}]}')
+    .withModel({ type: 'record', name: 'PublicKey', namespace: 'org.dsnp', fields: [{ name: 'publicKey', doc: 'Multicodec public key', type: 'bytes' }] })
```

Comment:
This was always supposed to be an object. Not sure if or why this might have worked before.